NSS-Alleged-Donkeys/WorkforceManagement
# Can Successfully Add an Employee (Attempt 2)

## Link to Ticket
closes # [HR Should be able to add a new employee](https://github.com/NSS-Alleged-Donkeys/WorkforceManagement/issues/2)

## Description of Proposed Changes
User can successfully add a new employee



## Steps to Test
In the CLI:
```
git fetch --all
git checkout MP2-PRAttempt2
```

Inside of Visual Studio:
Go to the `EmployeeTests.cs` inside of the `BangazonWorkforce.IntegrationTests`
Open the Test Explorer Window
Run the Test for `Post_CreateAddsEmployee`

After the Test passes
Run the project and go to `localhost:5000/Employee`
And a newly created Employee with a randomly generated First and Last Name should appear at the bottom of the list

## Impacted Areas in Application
`EmployeeTests.cs`
`EmployeeController.cs`
